showcase/radar-chart/radar-chart-with-tooltips.js

```diff
@@ -178,15 +123,28 @@ export default class RadarChartWithTooltips extends Component {
         colorRange={['transparent']}
         hideInnerMostValues={false}
         renderAxesOverPolygons={true}
+        numberOfGridlines={0}
       >
-        {hoveredCell &&
-          hoveredCell.dataName === 'Mercedes' && (
-            <Hint value={hoveredCell}>
-              <div style={tipStyle}>
-                {hoveredCell.domain}: {hoveredCell.value}
-              </div>
-            </Hint>
-          )}
+        {hoveredCell && (
+          <Hint value={hoveredCell}>
+            <div style={tipStyle}>
+              {hoveredCell.domain}: {hoveredCell.value}
+            </div>
+          </Hint>
+        )}
+        <RadarChartStraightGridLines
+          numberOfGridlines={5}
+          numberOfDomains={6}
+          width={450}
+          height={350}
+          margin={{top: 30, right: 60, left: 60, bottom: 30}}
+          style={{
+            marginTop: '-350px',
```

Comment:
This margin always needs to be a negative value of the height of the `RadarChart`. If approved, I would specifically call this out in the documentation.